packages/vue/src/examples/pages/thank-you/ThankYou.vue

```diff
@@ -2,70 +2,104 @@
   <div id="thank-you">
     <div class="banner">
       <div class="banner__info">
-        <SfHeading title="It's ordered!" class="sf-heading--no-underline" />
-        <p class="banner__order-number">
-          Order No. <strong>#80932031-030-00</strong>
-        </p>
+        <SfHeading
+          title="Thank You for your order!"
+          class="heading sf-heading--left"
+          :level="1"
+        />
+        <div class="banner__order-number">
+          <span>Order No.</span>
+          <strong>#80932031-030-00</strong>
+        </div>
       </div>
     </div>
     <div class="wrapper">
-      <SfHeading
-        title="You've successfully placed the order"
-        class="heading sf-heading--left"
-      />
-      <p class="paragraph">
-        You can check status of your order by using our delivery status feature.
-        You will receive an order confirmation e-mail with details of your order
-        and a link to track its progress.
-      </p>
-      <SfButton class="wrapper__notifications-button"
-        >ALLOW ORDER NOTIFICATIONS</SfButton
-      >
-      <div class="wrapper__contact">
+      <div class="wrapper__section">
         <SfHeading
+          title="Your Purchase"
+          class="heading--full sf-heading--left"
           :level="3"
-          class="heading--no-padding sf-heading--left sf-heading--no-underline"
-          title="Primary contacts for any questions"
-        ></SfHeading>
+        />
         <p class="paragraph">
-          <span>{{ address.name }}</span
-          ><br />
-          <span>{{ address.street }}</span
-          ><br />
-          <span>{{ address.city }}</span
-          ><br /><br />
-          <span>{{ address.email }}</span>
+          You have successfuly placed the order. You can check status of your
+          order by using our delivery status feature. You will receive an order
+          confirmation e-mail with details of your order and a link to track its
+          progress.
         </p>
-      </div>
-      <SfHeading title="Your Account" class="heading sf-heading--left" />
-      <p class="paragraph">
-        You can log to your account using e-mail and password defined earlier.
-        On your account you can edit your profile data, check history of
-        transactions, edit subscription to newsletter.
-      </p>
-      <SfHeading title="What can we improve" class="heading sf-heading--left" />
-      <p class="paragraph">
-        Your feedback is important to us. Let us know what we could improve.
-      </p>
-      <div class="wrapper__buttons">
-        <SfButton class="feedback-button color-secondary sf-button--full-width"
-          >SEND MY FEEDBACK</SfButton
+        <SfDivider class="first-divider" />
```

Comment:
I think divider should be visible on desktop only, see designs on mobile :)

Reply:
There was a divider before, apparently there was a change in designs ;)

Reply:
Removing it :)

Reply:
... I have one idea, maybe you can delate SfDivider and use border-width: 2px 0. Thanks for that you can remove
```
.first-divider {
  --divider-margin: var(--spacer-base) 0 var(--spacer-sm) 0;
  --divider-border-color: var(--c-white);
}

.second-divider {
  --divider-margin: var(--spacer-sm) 0 0 0;
  --divider-border-color: var(--c-white);
}
```